src/inference/variational.jl

```diff
@@ -140,7 +144,7 @@ function black_box_vi!(
             elbo_estimate += (log_weight / samples_per_iter)
 
             # accumulate the generative model gradients
-            _maybe_accumulate_param_grad!(model_trace, model_update)
+            _maybe_accumulate_param_grad!(model_trace, model_update, 1.0 / samples_per_iter)
```

Comment:
Previously, changing the number of samples changed the magnitude of the gradient and would have required a reciprocal change to the step size to maintain similar behavior. This fixes this.